frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -146,8 +152,7 @@ export default {
       if (!clusterRequiresZoneNetworkConfiguration) {
         return -1 // not applicable - no limit
       }
-      const totalNumberOfPossibleNetworkConfigurations = this.currentZonesWithNetworkConfigInShoot.length + this.currentFreeNetworks.length
-      if (totalNumberOfPossibleNetworkConfigurations >= this.allZones.length) {
+      if (this.currentFreeNetworks.length >= this.unusedZones.length) {
         return -1 // enough free networks - no limit
       }
```

Comment:
maybe something like to get rid of the comment and magic number
```js
const NO_LIMIT = -1

export default {
...

  const hasFreeNetworks = this.currentFreeNetworks.length >= this.unusedZones.length
  if (hasFreeNetworks) {
    return NO_LIMIT
  }
```

Reply:
done